aliases.sh

```diff
@@ -94,7 +95,7 @@ function try () {
 alias fixcamera="sudo killall VDCAssistant"
 # your IP in the local network
 # alias ipin="try -qe ipconfig getifaddr en0 "
-alias ipin="ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\.){3}[0-9]*).*/\2/p'"
+alias ipin="sudo ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\.){3}[0-9]*).*/\2/p'"
```

Comment:
If sudo is required in Linux, but not in mac, we could use an `if`, or maybe try it without sudo(silently) and if it does not work, try it again with sudo, like this:

```
function getIpIn() {
    ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\.){3}[0-9]*).*/\2/p'
}
alias ipin="getIpIn 2>/dev/null || sudo getIpIn"
```

This would avoid asking for sudo permission and password when it is not necessary

Reply:
We need to find another way...

```bash
alias ipin="getIpIn 2>/dev/null || sudo getIpIn"
```

This way when you need sudo previlegious,  case running whitout it,
we have not an error, so never will be executed by sudo